bf2fc6cc711aee1a0c2a/kas-fleet-manager
# Stratosphere

## Description

RHOSAK on AWS Marketplace related changes.
Jira: https://issues.redhat.com/browse/MGDSTRM-6922

## Verification Steps
1. For verification an organization with access to marketplace billing and linked cloud accounts is needed
2. Create a Kafka instance providing a valid cloud account using the new parameters `marketplace` and `billing_cloud_account_id`.
3. Make sure the instance is created and the two fields are persisted in the database
4. Make sure the fields are returned in the GET and LIST endpoints
5. Make sure edge cases work: omit marketplace, no billing account provided but only one linked etc.
